project0/index.js

```diff
@@ -0,0 +1,283 @@
+// implementation was based on the MVC pattern
+
+class QueueModel {
```

Comment:
Can we decompose this to two model. 
`PatientsQueueModel` and `ResolutionsModel`?